master/reference/node/configuration.md

```diff
@@ -144,3 +144,33 @@ e.g.
 IP_AUTODETECTION_METHOD=interface=eth.*
 IP6_AUTODETECTION_METHOD=interface=eth.*
 ```
+
+### Node readiness
```

Comment:
I'd organize this in reverse.

e.g.

### calico/node readiness

#### BGP and Policy readiness
Do this to enable readiness....

#### Policy-only readiness
Do _this_ to enable readiness...